test/test_http11.rb

```diff
@@ -183,4 +183,30 @@ def test_horrible_queries
     end
 
   end
+
+  def test_13632_workaround
```

Comment:
It's because the version of 2.5.0 on CI is very old and does not include the fix. Please, disregard.